### PR TITLE
Fix errors on boot

### PR DESCRIPTION
**Pull request by gjtorikian**
Should close https://github.com/skevy/graphiql-app/issues/82 and https://github.com/skevy/graphiql-app/issues/90.

When the app first launches, it immediately tries to grab a schema from an endpoint URL. But if there's no URL, the `fetch` function fails with an obscure error.

### CODE REVIEW

**Comment by zeke** [comment]
Works!

screen shot 2017-10-23 at 4 00 48 pm

**Comment by Skaronator** [comment]
Thanks for fixing! Could you publish a new version?

**Comment by gjtorikian** [comment]
Ah, yes, 0.6.3 was made yesterday, but not released on GitHub: https://github.com/skevy/graphiql-app/releases/tag/v0.6.3

And here's the cask update (pending as of this comment): https://github.com/caskroom/homebrew-cask/pull/39997

**Comment by Skaronator** [comment]
Awsome! Thanks for the quick response 👍

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
   "scripts": {
     "dev-server": "webpack-dev-server --config webpack/webpack-dev-server.config.js --progress --colors --port 2992 --inline",
     "hot-dev-server": "webpack-dev-server --config webpack/webpack-hot-dev-server.config.js --hot --progress --colors --port 2992 --inline",
+    "build-dev": "webpack --config webpack/webpack.config.js --progress --profile --colors",
```
Comment (AnthonyMeaux101) [inline package.json]: ``

Comment (AnthonyMeaux101) [inline package.json]: r

**Pull request merged**